### ISSUE TITLE
v0.10 migration issue

### ISSUE THREAD

**Issue by glennjacobs** [getcandy/candy-api#314]
Clean install of L8 on PHP 7.4

```
Migrating: 2020_09_07_150000_add_fields_to_countries_table
Migrated:  2020_09_07_150000_add_fields_to_countries_table (11.64ms)
Migrating: 2020_09_07_151640_add_states_table

   Illuminate\Database\QueryException

  SQLSTATE[23000]: Integrity constraint violation: 1452 Cannot add or update a child row: a foreign key constraint fails (`blah`.`states`, CONSTRAINT `states_country_id_foreign` FOREIGN KEY (`country_id`) REFERENCES `countries` (`id`)) (SQL: insert into `states` (`country_id`, `name`, `code`) values (15, Australian Capital Territory, ACT))

  at vendor/laravel/framework/src/Illuminate/Database/Connection.php:671
    667▕         // If an exception occurs when attempting to run a query, we'll format the error
    668▕         // message to include the bindings with SQL, which will make this exception a
    669▕         // lot more helpful to the developer instead of just the database's errors.
    670▕         catch (Exception $e) {
  ➜ 671▕             throw new QueryException(
    672▕                 $query, $this->prepareBindings($bindings), $e
    673▕             );
    674▕         }
    675▕

      +48 vendor frames
  49  artisan:37
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
```

**Comment by glennjacobs** [comment]
Seems the countries table is empty and it's trying to insert states that relate to a country.

**Comment by glennjacobs** [comment]
Yep, no migration inserts any countries. Countries are added via the `GetCandy\Api\Installer\Runners\CountryRunner`. Shouldn't states get added the same way?

**Comment by alecritson** [comment]
This should be resolved in `0.10.1-beta`
